### PR TITLE
cpdp-1143: landing page content reworked for private beta

### PR DESCRIPTION
**Pull request by nubz**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -37,52 +37,36 @@ unit-input.assistive.percent=do not enter the percent sign
...
-landingPage.whatYoullNeed.bullet8=property type, if you’re a non-resident
-landingPage.ifYoureAnAgent=If you’re an agent
-landingPage.ifYoureAnAgent.p1=Go to your <a href="{0}">agent services account</a> to access this service. If you do not already have one, you can <a href="{1}">apply for an agent services account</a>.
-landingPage.ifYoureRepresentingATrust=If you’re representing a trust
-landingPage.ifYoureRepresentingATrust.p1=The trust will need to be <a href="{0}">registered</a>. You will need the trust’s Self Assessment Unique Taxpayer Reference.
+landingPage.intro.p1=You may have to pay <a href="{0}">Capital Gains Tax</a> if you make a profit (‘gain’) when you sell (or ‘<a href="{0}">dispose of</a>’) property that’s not your home. For example, buy-to-let properties, land or inherited property.
+landingPage.intro.p2=In most cases you do not need to pay the tax when you <a href="{0}">sell your main home</a>.
+landingPage.createAccount.h2=Create a Capital Gains Tax on UK property account
+landingPage.createAccount.p1=Before 6 April 2020, you'll be able sign in to this service using your Government Gateway user ID and password to:
```
Comment (AndyHWChung) [inline conf/messages]: this looks like a straight quote in `you'll`

File: conf/messages [hunk 2]
```diff
@@ -37,52 +37,36 @@ unit-input.assistive.percent=do not enter the percent sign
...
+landingPage.createAccount.bullet1=create a Capital Gains Tax on UK property account
+landingPage.createAccount.bullet2=update your contact details in your Capital Gains Tax on UK property account once you have created one
+landingPage.createAccount.p2=If you do not have a Government Gateway user ID, you can create one.
+
+landingPage.howTo.h2=How to report and pay the tax
+landingPage.warning=From 6 April 2020, you will have 30 days after the property completion date to report and pay Capital Gains Tax on your UK property sales or disposals.
+landingPage.howTo.p1=From 6 April 2020, you will be able to use this service to:
+landingPage.howTo.list1.bullet1=report and pay any Capital Gains Tax due on UK property sales or disposals made from 6 April 2020
+landingPage.howTo.list1.bullet2=view and amend your Capital Gains Tax on UK property returns made from 6 April 2020
+landingPage.howTo.p2=If you’re a non-resident, you'll also be able to use this service from 6 April 2020 to report the sale or disposal and pay the tax on:
```
Comment (AndyHWChung) [inline conf/messages]: same thing in this `this looks like a straight quote in you'll`

Comment (nubz) [reply conf/messages]: yep replaced

File: conf/messages (hunk 1 above)
Comment (nubz) [reply conf/messages]: yep replaced

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**